recipes/swi-prolog/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "swi-prolog" %}
+{% set version = "7.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #url: https://github.com/SWI-Prolog/swipl-devel/archive/V7.7.0.tar.gz
+  # Makes use of git_url as it needs submodules from git, not available in archive
+  git_url: https://github.com/SWI-Prolog/swipl-devel.git
+  git_rev: V{{ version }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - curl
+    - openjdk
+    - autoconf
+    - gcc   # [not osx]
+    - gmp
+    - llvm  # [osx]
+    - libgcc  # [not osx]
+    - ncurses
```

Comment:
nnucrese needs to be pinned, same for zlib and gmp I assume

Reply:
is there any variable for pinning those common libs ? Can't find in conda forge doc for pinning